hail_search/queries/snv_indel.py

```diff
@@ -25,6 +27,23 @@ class SnvIndelHailTableQuery(SnvIndelHailTableQuery37):
         ('is_gt_10_percent', 0.1),
     ])
 
+    def _get_allowed_consequence_ids(self, annotations):
+        consequence_ids = super()._get_allowed_consequence_ids(annotations)
+        if EXTENDED_SPLICE_REGION_CONSEQUENCE in (annotations.get('extended_splice_site') or []):
+            consequence_ids.add(EXTENDED_SPLICE_REGION_CONSEQUENCE)
+        return consequence_ids
+
+    @staticmethod
+    def _get_allowed_transcripts_filter(allowed_consequence_ids):
+        has_extended_splice = EXTENDED_SPLICE_REGION_CONSEQUENCE in allowed_consequence_ids
+        allowed_consequence_ids = allowed_consequence_ids - {EXTENDED_SPLICE_REGION_CONSEQUENCE}
```

Comment:
why do we need to remove 'extended_intronic_splice_region_variant' from the ids here?

Reply:
because everything else in here is an integer ID and so if we leave this string in when it gets converted to a hail set it will have the wrong type